web/src/components/ChatController.tsx

```diff
@@ -37,11 +37,11 @@ const ChatController = ({ city, language }: ChatControllerProps): ReactElement =
   const isBrowserTabActive = useIsTabActive()
 
   useEffect(() => {
-    if (!isBrowserTabActive) {
-      return undefined
+    if (isBrowserTabActive) {
```

Comment:
I liked it the over way round more. Early exit is clearer and avoids nested if/elses if more complexity is added to this function in the future

Reply:
I get the point, but as long as there is only this little complexity, I prefer this as no negation is needed. When additional complexity is added, this can be changed again if necessary.